Extension/c_cpp_properties.schema.json

```diff
@@ -239,7 +250,6 @@
         },
         "enableConfigurationSquiggles": {
             "type": "boolean",
-            "default": true,
```

Comment:
Why is this default being removed?

Reply:
[see comment by @colengms](https://github.com/microsoft/vscode-cpptools/pull/12960#discussion_r1894413496)

Reply:
My mistake. I thought this was the setting in `package.json`.